Amethyst/Managers/WindowManager.swift

```diff
@@ -334,8 +334,14 @@ final class WindowManager<Application: ApplicationType>: NSObject, Codable {
         guard let screen = window.screen() else {
             return
         }
+        let windowIDsArray = CGWindowsInfo.windowIDsArray(window)
+        guard let spaces = CGSCopySpacesForWindows(CGSMainConnectionID(), kCGSAllSpacesMask, windowIDsArray)?.takeRetainedValue() else {
+            return
+        }
+
+        let space = (spaces as NSArray as? [NSNumber])?.first?.intValue
```

Comment:
Sorry, I meant that you should take this entire chunk and make it a function in CGInfo.

Reply:
> Sorry, I meant that you should take this entire chunk and make it a function in CGInfo.

Ok, I've been using a block application to control my computer, I only have one hour each day, hope I will do it correctly this night.😆